repositories/analytics_views/users.sql

```diff
@@ -0,0 +1,15 @@
+DROP VIEW IF EXISTS analytics.users;
+
+CREATE VIEW
+      analytics.users
+WITH
+      (security_invoker = false) AS
+SELECT
+      id,
+      organization_id,
+      email,
+      firebase_uid,
+      role,
+      deleted_at
```

Comment:
Do we want to expose all user data ?

Reply:
IMO anything that goes to segment is fair game to see on metabase. In particular I'd not be too stressed about the email. The firebase uid, I don't know, I don't think it's much more confidential but then again it's kinda useless to know it so fine to remove it from here